test/stresstest/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Fjage = "e0fd600c-be67-11e9-1f90-d366689e4029"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
Likely to have problems with CI with GLMakie? Do we need it for tests?

Reply:
This is in a separate `test/ >>> stresstest <<< /Project.toml` file, not `test/Project.toml`. So this won't be loaded as part of CI.